rtmbot.py

```diff
@@ -153,26 +161,71 @@ def parse_args():
     )
     return parser.parse_args()
 
+def flag_is_true(flag_str):
+    """ Checks if a string flag should evaluate to True. """
+    flag_str = flag_str.strip().lower()
+    return flag_str == 'true'
+
+def invoke():
+    """ Starts the bot. The entry point into the app. """
+    global config
+    global debug
 
-if __name__ == "__main__":
     args = parse_args()
-    directory = os.path.dirname(sys.argv[0])
-    if not directory.startswith('/'):
-        directory = os.path.abspath("{}/{}".format(os.getcwd(),
-                                directory
-                                ))
-
-    config = yaml.load(file(args.config or 'rtmbot.conf', 'r'))
-    debug = config["DEBUG"]
+
+    # Try to load config from a specified file, or rtmbot.conf if
+    # it exists in the working directory.
+    using_file_config = False
+    if args.config:
+        using_file_config = True
+        config = yaml.load(file(args.config))
+    elif os.path.isfile('rtmbot.conf'):
+        using_file_config = True
+        config = yaml.load(file('rtmbot.conf'))
```

Comment:
Hmm, that's a good point. I think starting from a clean slate, it would be better to use a YAML extension. Then again, I am hesitant to have multiple default names for the configuration file, that just feels like more of a mess than something so simple is worth. I updated the README to make it clear that it's a YAML file though, hopefully that will help.